custom/layouts/partials/head.ejs

```diff
@@ -0,0 +1,25 @@
+<head>
+    <meta charset="utf-8">
+    <meta id="meta" name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title><%= title %></title>
+    <!-- HTML5 shim, for IE6-8 support of HTML elements -->
+    <!--[if lt IE 9]>
+      <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
+    <![endif]-->
+  
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+  
+    <link rel="stylesheet"
+        href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/10.0.0/styles/default.min.css">
+  
+    <% if (locals.inlineCSS) { %>
+      <style type="text/css"><%- inlineCSS %></style>
+    <% } else { %>
```

Comment:
how does the CSS get included now? seems you removed the original `style.css` reference but not replaced with anything

Reply:
I'm not sure what this inlineCSS is.  It uses the generated CSS even on the old version.  I'll have to investigate.

Reply:
Well, it looks like this comment explains it all:

```
// Because asset requests are authenticated, we inline the images and CSS
// that we need to serve to logged-out users, e.g. on the auth error page
```